src/api/models/article.js

```diff
@@ -16,7 +16,7 @@ export default (sequelize, DataTypes) => {
       title: { type: DataTypes.STRING, allowNull: false },
       description: { type: DataTypes.TEXT, allowNull: false },
       body: { type: DataTypes.TEXT, allowNull: false },
-      slug: { type: DataTypes.STRING, allowNull: false },
+      slug: { type: DataTypes.STRING, allowNull: false, unique: true },
```

Comment:
Why did you add unique on slug yet the algorithm for it is to make it unique ?

Reply:
we need to make sure that the database reflects the reality